ruby/lib/minitest/queue/grind_recorder.rb

```diff
@@ -45,9 +45,9 @@ def record(test)
 
         stats = self.class.counters
         if (test.failure || test.error?) && !test.skipped?
-          build.record_error("#{test.klass}##{test.name}##{SecureRandom.uuid}", dump(test), stats: stats)
```

Comment:
@mcgain I think this still has to be included. I'm going to play around with this.